token/sdk/tms/tms.go

```diff
@@ -41,13 +39,26 @@ func (p *PostInitializer) PostInit(tms driver.TokenManagerService, networkID, ch
 		Channel:   channel,
 		Namespace: namespace,
 	}
+
 	// restore owner db
 	if err := p.ownerManager.RestoreTMS(tmsID); err != nil {
 		return errors.WithMessagef(err, "failed to restore onwer dbs for [%s]", tmsID)
 	}
+
 	// restore auditor db
 	if err := p.auditorManager.RestoreTMS(tmsID); err != nil {
 		return errors.WithMessagef(err, "failed to restore auditor dbs for [%s]", tmsID)
 	}
+
+	// set supported tokens
+	tokens, err := p.tokensProvider.Tokens(tmsID)
+	if err != nil {
+		return errors.WithMessagef(err, "failed to get tokens for [%s]", tmsID)
+	}
+	supportedTokens := tms.TokensService().SupportedTokenTypes()
```

Comment:
Would it be maybe better to remove `SetSupportedTokenTypes` and instead get the supported token types on the fly in the token service? This way we can update them only in one place.